src/android/AccessibilityNativeUtil.ts

```diff
@@ -1,5 +1,5 @@
 /**
-* AccessibilityUtil.ts
+* AccessibilityNativeUtil.ts
```

Comment:
Can't this still be called AccessibilityUtil (Native is redundant since it's in the android folder